Feat/ping

Ose-4g opened the pull request:
Created endpoint /ping to return json data telling the state of the server. 

live test endpoint = https://goals.zuri.chat/api/v1/ping

It fixes issue #357

Ose-4g commented:
@Asylcreek  I've made a new pull request

Asylcreek commented:
Hey man. Question: Are you running the server in production mode? 'Cause the log files are supposed to be empty.

And please change the "api/v1/ping" to "/ping", this is because the frontend is not consuming this route.

And please pull from the staging branch before your next push, there are some updates your code doesn't have.

Ose-4g commented:
Alright @Asylcreek I've made the changes. 
- Removed all logs from the error files
- Changed my NODE_ENV to development locally
- Changed the endpoint to /ping
- Pulled updates from upstream staging to avoid merge conflicts

Asylcreek commented on app.js:
```diff
@@ -46,6 +47,9 @@ app.use(compression());
 // Api routes
 app.use('/api/v1/goals', rateLimiter(), goalRouter);
+app.use('/ping', rateLimiter(), pingRouter);
+app.use('/api/v1/info', rateLimiter(), pluginInfoRouter);
```
Please delete this line

Asylcreek commented on app.js:
```diff
@@ -46,6 +47,9 @@ app.use(compression());
 app.use('/api/v1/goals', rateLimiter(), goalRouter);
+app.use('/ping', rateLimiter(), pingRouter);
+app.use('/api/v1/info', rateLimiter(), pluginInfoRouter);
+app.use('/api/v1/sidebar', rateLimiter(), sidebarRouter);
```
Please delete this line

Ose-4g replied:
Alright. So I shouldn't use a router for the endpoint ???

Ose-4g commented:
Alright @Asylcreek  I've updated the document as requuested.
